PUBLISHING.md

```diff
@@ -0,0 +1,111 @@
+## Publishing a New Release
+
+### Testing
+
+It's important to make sure that the release has been thoroughly tested before it's published. This will get added to over time but at a minimum, we should make sure basics of all modules work without issue.
+
+- Custom Status: New posts are set to default custom status (for both Block and Classic Editors).
+- Custom Status: Status can be changed to another custom one (for both Block and Classic Editors).
+- Custom Status: Publishing and unpublishing works (for both Block and Classic Editors).
+- Custom Status: Enabling and disabling the feature works as expected (for both Block and Classic Editors).
+- Editorial Comments: Can add comments and reply to them (for both Block and Classic Editors).
+- Notifications: Can follow and unfollow posts (works for both Block and Classic Editors).
```

Comment:
Maybe also:
 - Notifications:  Taking action on a post (saving, adding Editorial comment, etc) will auto-subscribe you to the post